### DISCUSSION TITLE
make from root dir

### DISCUSSION THREAD

**Discussion by sethahrenbach** [AU-COBRA/ConCert#211] [Q&A]
The README in the root dir prescribes running `make`, however this throws an error due to missing makefiles. I want to confirm that this is a generic instruction to be run from within each subproject; is that correct?

**Comment by sethahrenbach** [comment]
Actually I discovered that re-running `eval $(opam env)` fixes this. I had previously cloned and performed the setup, but needed to re-run that command now, and `make` works from root dir.

**Reply by 4ever2** [reply]
You need to re-run `eval $(opam env)` every time you open a new terminal unless you have configured your terminal to automatically set up the opam environment variables.
